### DISCUSSION TITLE
Termux install

### DISCUSSION THREAD

**Discussion by b9Joker108** [shadowsocks/shadowsocks-rust#1459] [Q&A]
I have successfully installed shadowsocks-rust in Termux with cargo and added the cargo bin to the $PATH environment variable.

I can't locate a json config file for shadowsocks-rust, do I just create one? What should it be called? Is there a template of what it should contain? Where is it best to save it? 

I read: https://shadowsocks.org/doc/configs.html but it didn't answer any of
My questions. I also found: https://cloudzy.com/blog/shadowsocks-config/ informative, but it also didn't answer my questions.

Sorry to be a hassle, but I am a noob and cannot find any information specific to Termux either with a browser search, YouTube search, or in discussions or issues in this repository, or in the documentation in the initial repository.

I could really do with assistance

**Comment by zonyitoo** [comment]
> I can't locate a json config file for shadowsocks-rust, do I just create one? What should it be called? Is there a template of what it should contain? Where is it best to save it?

You can create a configuration file with whatever name you want in whatever location you want it to be.
